packages/smooth_app/lib/database/dao_folksonomy.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:smooth_app/database/abstract_sql_dao.dart';
+import 'package:smooth_app/database/local_database.dart';
+import 'package:sqflite/sqflite.dart';
+
+class DaoFolksonomy extends AbstractSqlDao {
+  DaoFolksonomy(super.localDatabase);
+
+  static const String _table = 'folksonomy';
+  static const String _columnBarcode = 'barcode';
+  static const String _columnTags = 'tagsString';
+  static const String _columnLastUpdateTimestamp = 'lastUpdateTimeStamp';
```

Comment:
In SQL databases we typically name columns and tables with underscores, something like `last_update`.
Never mind, it's a pain in the neck to change column names and nobody cares about it.